root/static/scripts/common/coverart.js

```diff
@@ -20,8 +20,9 @@ $(function () {
         if ($e.data('fallback') && $e.attr('src') === thumbnailUrl) {
           $e.attr('src', $e.data('fallback'));
         } else {
-          $e.closest('a')
-            .replaceWith('<em>' + $e.data('message') + '</em>');
+          $e.closest('a').replaceWith(
+            '<em class="cover-art-error">' + $e.data('message') + '</em>',
```

Comment:
Thank you! Just one suggestion (you will have to `import he from 'he';`).
```suggestion
            '<em class="cover-art-error">' + he.escape($e.data('message')) + '</em>',
```

Reply:
Thanks, I'll push a squashed change with this suggestion.

Reply:
I'm very sorry, I just realized this is wrong because the message can actually contain HTML (I thought it was just plain text). So let's just revert back to 780e9a9.

Reply:
No problem -- thanks for switching it back!